lib/Common/Memory/Recycler.cpp

```diff
@@ -1298,6 +1314,28 @@ bool Recycler::AllowNativeCodeBumpAllocation()
 }
 
 
+#if DBG
+void Recycler::VerifyPageHeapFillAfterAlloc(char* memBlock)
+{
+    if (IsPageHeapEnabled())
+    {
+        HeapBlock* heapBlock = this->FindHeapBlock(memBlock);
+        if (heapBlock->IsLargeHeapBlock())
+        {
+            LargeHeapBlock* largeHeapBlock = (LargeHeapBlock*)heapBlock;
+            if (largeHeapBlock->InPageHeapMode())
```

Comment:
added assert

---
In reply to: [60989873](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60989873) [](ancestors = 60989873)